### ISSUE TITLE
command aliasing -- "plotsbot fto" / "plotsbot, first timers issues?"

### ISSUE THREAD

**Issue by jywarren** [publiclab/plotsbot#75]
I'd like to propose a way we could make different triggers for behaviors to cover various ways of asking questions. Even in different languages! So different trigger words could cause the same behavior to run.

This could be done either inside a behavior, by making a list of triggers, or by making a "wrapper" behavior that then just runs a different behavior.

**Comment by ryzokuken** [comment]
@jywarren Perhaps we could give an array of strings instead of a single one for the message trigger keyword?

**Comment by jywarren** [comment]
yeah that sounds good. Is it documented anywhere that it searches for the
string through whole messages to make matches? Sorry, can you point me at
the documentation for trigger keywords? Thanks!

**Comment by ryzokuken** [comment]
@jywarren take a look at pt. 3 in https://github.com/publiclab/plotsbot#behavior-model
